roles/contrail_master/tasks/main.yaml

```diff
@@ -0,0 +1,50 @@
+---
+- when:
+  - docker_registry is not defined
+  - inventory_hostname == contrail_first_master
+
+  block:
+  - name: Untar Contrail tar-file
```

Comment:
OpenContrail images are now available on Docker Hub, can we pull these if users don't want to download the RHEL7 tarballs from Juniper's website (particularly for Origin deployments which may be done on CentOS).